integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDomainInPV.java

```diff
@@ -324,6 +332,49 @@ public void testIstioDomainHomeInPv() {
     boolean checkApp = checkAppUsingHostHeader(url, domainNamespace + ".org");
     assertTrue(checkApp, "Failed to access WebLogic application");
 
+    // Refer JIRA https://jira.oraclecorp.com/jira/browse/OWLS-86407
+    // Stop and Start the managed server in absense of administration server
+    assertTrue(patchServerStartPolicy(domainUid, domainNamespace,
+         "/spec/adminServer/serverStartPolicy", "NEVER"),
+         "Failed to patch administrationi server serverStartPolicy to NEVER");
+    logger.info("Domain is patched to shutdown administration server");
+    checkPodDeleted(adminServerPodName, domainUid, domainNamespace);
+    logger.info("Administration server shutdown success");
+
+    boolean scalingSuccess = assertDoesNotThrow(() ->
+        scaleCluster(domainUid, domainNamespace, "cluster-1", 1),
+        String.format("Scaling down cluster cluster-1 of domain %s in namespace %s failed",
+        domainUid, domainNamespace));
+    assertTrue(scalingSuccess,
+        String.format("Cluster scaling failed for domain %s in namespace %s", domainUid, domainNamespace));
+    logger.info("Cluster is scaled down in absense of administration server");
+    checkPodDeleted(managedServerPodNamePrefix + "2", domainUid, domainNamespace);
+    logger.info("Managed Server stopped in absense of administration server");
+
+    scalingSuccess = assertDoesNotThrow(() ->
+        scaleCluster(domainUid, domainNamespace, "cluster-1", 2),
+        String.format("Scaling up cluster cluster-1 of domain %s in namespace %s failed",
+        domainUid, domainNamespace));
+    assertTrue(scalingSuccess,
+        String.format("Cluster scaling failed for domain %s in namespace %s", domainUid, domainNamespace));
+    logger.info("Cluster is scaled up in absense of administration server");
+    checkServiceExists(managedServerPodNamePrefix + "2", domainNamespace);
+    checkPodReady(managedServerPodNamePrefix + "2", domainUid, domainNamespace);
+    logger.info("Managed Server started in absense of administration server");
+  }
+
+  private void checkPodDeleted(String podName, String domainUid, String domNamespace) {
```

Comment:
Check CommonTestUtils for this method and call that.

Reply:
updated tests to use CommonTestUtils .checkPodDeleted()